js_modules/dagit/packages/core/src/instance/BackfillRow.tsx

```diff
@@ -70,10 +75,15 @@ export const BackfillRow = ({
     },
   );
 
+  const statusUnsupported = backfill.numPartitions === null || backfill.partitionNames === null;
+
   // Note: We switch queries based on how many partitions there are to display,
   // because the detail is nice for small backfills but breaks for 100k+ partitions
-  const [queryStatus, queryResult] =
-    backfill.numPartitions > BACKFILL_PARTITIONS_COUNTS_THRESHOLD ? statusCounts : statusDetails;
+  const [queryStatus, queryResult] = statusUnsupported
```

Comment:
This is confusing -- `queryStatus` is the _function_, and `queryResult` is the data? These variable names are awfully similar. Could they be renamed to clarify this? Since the values are separated from the `useLazyQuery` calls themselves, it's a little tough to understand what these values actually are.

Reply:
Yeah good call - I renamed this `[statusQueryFn, statusQueryResult]` which seems better!